From 7a24ce9f5045ac6faf66a08bbb814f3a82c919b1 Mon Sep 17 00:00:00 2001
From: Sergey Matyukevich <geomatsi@gmail.com>
Date: Fri, 19 May 2017 23:39:48 +0300
Subject: [PATCH] ARM: dts: orange-pi-zero: specify XR819 WiFi chip interrupts

The orange-pi-zero board has Allwinner XR819 SDIO wifi chip. The board
dts file provides a node enabling mmc1 controller. Together with an
out-of-tree driver https://github.com/fifteenhex/xradio this node
enables using the wifi chip.

This commit specifies XR819 interrupt configuration for the driver.

Signed-off-by: Sergey Matyukevich <geomatsi@gmail.com>
---
 arch/arm/boot/dts/allwinner/sun8i-h2-plus-orangepi-zero.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/allwinner/sun8i-h2-plus-orangepi-zero.dts b/arch/arm/boot/dts/allwinner/sun8i-h2-plus-orangepi-zero.dts
index 2a45770f7b83..667038973bb7 100644
--- a/arch/arm/boot/dts/allwinner/sun8i-h2-plus-orangepi-zero.dts
+++ b/arch/arm/boot/dts/allwinner/sun8i-h2-plus-orangepi-zero.dts
@@ -152,6 +152,9 @@ &mmc1 {
 	 */
 	xr819: wifi@1 {
 		reg = <1>;
+		compatible = "xradio,xr819";
+		interrupt-parent = <&pio>;
+		interrupts = <6 10 IRQ_TYPE_EDGE_RISING>;
 	};
 };
 
-- 
2.47.1

